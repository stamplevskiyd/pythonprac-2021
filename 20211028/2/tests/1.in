print(*list(slide("ABCDE", 3)))
