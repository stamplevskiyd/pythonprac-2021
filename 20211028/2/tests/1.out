A B C B C D C D E D E E
